flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/KinesisPartitioner.java

```diff
@@ -18,6 +18,7 @@
 package org.apache.flink.streaming.connectors.kinesis;
```

Comment:
This should be moved to the new maven module?

Reply:
It is only left here for backward compatibility (i.e. customers implementing the class). I believe moving it will be incompatible.